### ISSUE TITLE
Repeatable form (group) fieldtype

### ISSUE THREAD

**Issue by TimGeyssens** [umbraco/Umbraco.Forms.Issues#325]
Not sure how this could be done, but what would be a nice addition is to make repeatable fields (or group of fields)

Let say for an event attendees, you don't know in advance how many people they sign up...

So have name + email being repeatable would solve this...

**Comment by nul800sebastiaan** [comment]
Thanks for your feature request. We're working on fixing bugs for the foreseeable future, but we'll put this on the list for when we are ready to work on new fieldtypes. 

For now, you can implement your own custom fieldtypes to achieve this functionality. https://our.umbraco.com/documentation/add-ons/umbracoforms/developer/Extending/Adding-a-Fieldtype

**Comment by TimGeyssens** [comment]
Indeed, but not sure how to handle the data storage for a field (group) like this.... since it will possible need some refactoring of the forms core
